packages/documentation/src/content/docs/telemetry/integrating.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Building on Top of Rafiki Metrics
+---
+
+Rafiki allows for integrating [Account Servicing Entities](/reference/glossary#account-servicing-entity) (ASE) to build their own telemetry solution based on the [OpenTelemetry](https://opentelemetry.io/) standardized metrics format that Rafiki exposes.
+
+In order to do so, the integrating ASE must deploy its own OpenTelemetry collector that should act as a sidecar container to Rafiki. It needs to provide the OpenTelemetry collector's ingest endpoint so that Rafiki can start sending metrics to it.
+
+## Rafiki Telemetry Environment Variables
+
+- `ENABLE_TELEMETRY`: boolean, defaults to `true`.
+- `OPEN_TELEMETRY_COLLECTOR_URL`: CSV of URLs (e.g., `http://otel-collector-NLB-e3172ff9d2f4bc8a.elb.eu-west-2.amazonaws.com:4317,http://happy-life-otel-collector:4317`).
+- `OPEN_TELEMETRY_EXPORT_INTERVAL`: number in milliseconds, defaults to `15000`.
+- `TELEMETRY_EXCHANGE_RATES_URL`: string URL, defaults to `https://telemetry-exchange-rates.s3.amazonaws.com/exchange-rates-usd.json`. If set, the response format of the external exchange rates API should be of type Rates, as the rates service expects.
+
+## Example Docker OpentTelemetry Collector Image and Configuration
+
+Example of Docker OpenTelemetry Collector image and configuration that integrates with Rafiki and sends data to a Prometheus remote write endpoint:
+
+(it can be tested in our [Local Playground](/playground/overview) setup, by also providing the environment variables listed above to happy-life-backend in the [docker-compose](https://github.com/interledger/rafiki/blob/main/localenv/happy-life-bank/docker-compose.yml))
+
+#### Docker-compose config:
+
+```yaml
+#Serves as example for optional local collector configuration
+happy-life-otel-collector:
+  image: otel/opentelemetry-collector-contrib:latest
+  command: ['--config=/etc/otel-collector-config.yaml', '']
+  environment:
+    - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID-''}
+    - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY-''}
+  volumes:
+    - ../collector/otel-collector-config.yaml:/etc/otel-collector-config.yaml
+  networks:
+    - rafiki
+  expose:
+    - 4317
+  ports:
+    - '13132:13133' # health_check extension
+```
+
+#### OpenTelemetry OTEL collector config:
```

Comment:
maybe worth linking to https://opentelemetry.io/docs/collector/configuration/ in this section?